src/ripple/app/main/Application.cpp

```diff
@@ -1339,8 +1339,6 @@ ApplicationImp::setup()
                 {
                     // Fall back to syncing from the network, such as
                     // when there's no existing data.
-                    if (startUp == Config::NETWORK && !config_->standalone())
-                        m_networkOPs->setNeedNetworkLedger();
```

Comment:
Yeah, it's unrelated to levelization. It's related in the sense that it was introduced by the same PR as the levelization issue. And since it's completely unreachable, I figured it would safe to slip in.

Reply:
I looked at this just a little it harder.  I completely agree with @ximinez, that the removed code was unreachable.  So I'm giving this pull request an unreserved :+1:.  Thanks for taking care of this @ximinez!